src/maintainer/knowledge_base.rst

```diff
@@ -1179,6 +1179,24 @@ For now, you will have to add ``nvcuda.dll`` to the ``missing_dso_whitelist``::
       - "*/nvcuda.dll"   # [win]
 
 
+My feedstock is not building old CUDA versions anymore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With the `addition of CUDA 11.1 and 11.2 <https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1162>`_,
+the default build matrix for CUDA versions was trimmed down to versions 10.2, 11.0, 11.1, 11.2. If you want to
+re-add support for 9.2, 10.0 and 10.1, you need to follow these steps below:
+
+1. Rename the file ``.ci_support/migrations/cuda111_112.yaml`` to ``.ci_support/migrations/cuda111_112.disabled``.
+2. Add the contents of `this file <https://github.com/conda-forge/nvcc-feedstock/blob/master/recipe/conda_build_config.yaml>`_
+   to your ``recipe/conda_build_config.yaml``. Create it if it didn't exist yet.
+3. Re-render the feedstock.
+
+Note that by following this procedure you are opting out of the migration process for CUDA builds and you will have to
+update the contents of ``conda_build_config.yaml`` by hand. If you experience re-rendering errors, check that the contents
+of this file match the one present in the `nvcc-feedstock <https://github.com/conda-forge/nvcc-feedstock>`_. When in doubt,
+remove ``conda_build_config.yaml``, rename back the migrator file (step 1) and re-render.
```

Comment:
@beckermr it's probably better to mention how to switch between cos6/cos7 here, at least give an example.

Reply:
Yup. I plan to clean up these docs once the PR on smithy is done. Thanks for the pointer!

Reply:
my typo: give an example -> link to an example